client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -102,6 +95,49 @@ export default function WPCheckoutOrderSummary() {
 	);
 }
 
+function CheckoutSummaryFeaturesList() {
+	const hasDomainsInCart = useHasDomainsInCart();
+	const domains = useDomainsInCart();
+	const hasPlanInCart = useHasPlanInCart();
+	const translate = useTranslate();
+
+	const supportText = hasPlanInCart
+		? translate( 'Email and live chat support' )
+		: translate( 'Email support' );
+
+	let refundText = translate( 'Money back guarantee' );
+	if ( hasDomainsInCart && ! hasPlanInCart ) {
+		refundText = translate( '4 day money back guarantee' );
+	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
+		refundText = translate( '30 day money back guarantee' );
+	}
+
+	return (
+		<CheckoutSummaryFeaturesListUI>
+			{ hasDomainsInCart &&
+				domains.map( ( domain ) => {
+					return (
+						<CheckoutSummaryFeaturesListItem>
+							<WPCheckoutCheckIcon />
```

Comment:
There's a lot nested in here, including several levels of curly braces. There's nothing wrong with this but it might be a good candidate to move some things to their own component to make it easier to read and reason about. One component for each item might help? Then this block just becomes `domains.map( (domain) => <CheckoutSummaryFeaturesListItem item={domain} key={domain.id} /> )`